### PR TITLE
Newsitem GPT

### PR DESCRIPTION
**Pull request by Sch-Da**
added Arash's suggestions on picture names

### CODE REVIEW

File: content/news/2024-09-02-ChatGPT/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Another application example is the direct retrieval of research data from Zenodo in Galaxy. For example, published interviews can be imported directly and prepared for ChatGPT. Using sentiment analysis, the LLM analyses the interviewees' attitudes, groups them and extracts suitable passages from the material. The output can then be cleaned up with Galaxy’s other text processing tools or in the interactive environment of [Jupyter Notebooks](https://usegalaxy.eu/?tool_id=interactive_tool_jupyter_notebook), which is another tool in Galaxy.
+
+
+## Sharing your analysis and data
+
+The compiled workflow, the individual analysis steps and the data of each sub-step can be shared and repeated at any time for good scientific practice, simplifying research data management and reinforcing FAIR analysis. With the help of the [Galaxy Training Network](https://training.galaxyproject.org/), users can create an open-source tutorial generated out of their workflow to explain and share their analysis with other users.
```
Comment (bgruening) [inline content/news/2024-09-02-ChatGPT/index.md:53]: Maybe here or above I would embed one of your workflows here ... see here for an example: https://galaxyproject.org/news/2024-04-26-workflows-workflows-workflows/

File: content/news/2024-09-02-ChatGPT/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+| GPT-4 Turbo and GPT-4 | The previous set of high-intelligence models, Turbo with vision capabilities |
+| GPT-3.5 Turbo | A fast, inexpensive model for simple tasks |
+
+You can also combine the ChatGPT tool in Galaxy with the other tools within a more complex workflow:
+
+You could, for example, upload audio and video files from various sources to be automatically converted into text in Galaxy using [Whisper](https://usegalaxy.eu/?tool_id=whisper). After a short cleaning, ChatGPT can summarise the text, as visualised below.
+
+![ChatGPT tool](https://github.com/user-attachments/assets/bd1a82d5-be79-464f-8f57-9fe5487e2abf)
+
+The screenshot shows the integration of ChatGPT as a tool in Galaxy. The penal on the right side, the ‘History’, shows an example workflow. Here, a mp3-recording from Martin Luther King’s ‘I have a dream’ was converted into text, cleaned, and translated using ChatGPT. The prompt was adapted from [Prompting ChatGPT for Translation](https://doi.org/10.48550/arXiv.2403.00127).
```
Comment (neoformit) [inline content/news/2024-09-02-ChatGPT/index.md:44]: "penal" -> "panel"

File: content/news/2024-09-02-ChatGPT/index.md (hunk 1 above)
Comment (Sch-Da) [reply content/news/2024-09-02-ChatGPT/index.md:53]: great, working on it

File: content/news/2024-09-02-ChatGPT/index.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ tags: [tool]
...
+To help do so, [GPT](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/chatgpt_openai_api/chatgpt_openai_api/2024+galaxy1) is now available on [Galaxy](https://usegalaxy.eu/), an open-source platform for FAIR data analysis. This enables users to incorporate the LLM into more complex, automatable workflows and analyses. Galaxy Europe contains over 3,500 tools that users can combine in workflows to cover digital analysis without programming skills or expensive hardware. Users can share all steps and results of the analysis according to the FAIR principles, facilitating high reproducibility, transparency and efficient research data management.
```
Comment (bgruening) [inline content/news/2024-09-02-ChatGPT/index.md:14]:
```suggestion
To help do so, [GPT](https://usegalaxy.eu/root?tool_id=chatgpt_openai_api) is now available on [Galaxy](https://usegalaxy.eu/), an open-source platform for FAIR data analysis. This enables users to incorporate the LLM into more complex, automatable workflows and analyses. Galaxy Europe contains over 3,500 tools that users can combine in workflows to cover digital analysis without programming skills or expensive hardware. Users can share all steps and results of the analysis according to the FAIR principles, facilitating high reproducibility, transparency and efficient research data management.
```

Your link is correct and more precise. We call it the long tool ID. However, in this context you probably want to link always to the latest version, and for that you can use the short tool ID.

File: content/news/2024-09-02-ChatGPT/index.md (hunk 1 above)
Comment (Sch-Da) [reply content/news/2024-09-02-ChatGPT/index.md]: embedded one but would be glad for a feedback as the iframe looks bad in the preview and I am unsure how to best display the embedded workflow here.

**Comment by bgruening** [comment]
You could build and deploy the website locally to see all your changes - but it might be hard on Windows. I will check for you.

**Comment by Sch-Da** [comment]
That's very appreciated, thank you!

**Comment by bgruening** [comment]
image


LGTM!

**Pull request merged**